app/scripts/controllers/preferences.test.js

```diff
@@ -869,21 +869,21 @@ describe('preferences controller', function () {
       );
     });
   });
-  describe('setUseStaticTokenList', function () {
+  describe('setUseTokenDetection', function () {
     it('should default to false', function () {
       const state = preferencesController.store.getState();
-      assert.equal(state.useStaticTokenList, false);
+      assert.equal(state.useTokenDetection, true);
```

Comment:
This test should either be renamed... or, and I think its this one, the value should still be false?

Reply:
I renamed the test, the default value should be true.